ci/prototype.yml

```diff
@@ -21,9 +21,7 @@ run_prototype_test:
   stage: test
   image: ${PERSIST_IMAGE_NAME}
   script:
-    # here should go the commands to run the actual applications, `ls` is just to show what is already in the container, to get an idea
-    - ls -lh /src
-    - ls -lh --recursive /usr/local/
+    - (cd /src/integration && ./test_sample_mock.py)
```

Comment:
Commands are chained anyways, for better overview I would do one command at a time, i.e.
```
- cd /src/integration
- ./test_sample_mock.py
```